src/app/pages/applications/chart-releases/chart-releases.component.html

```diff
@@ -26,10 +26,15 @@
         </div>
 
         <div class="chart-button-row" fxLayout="row" fxLayoutAlign="end end" (click)="$event.stopPropagation()">
-          <button mat-button (click)="portal(item)" *ngIf="item.status === ChartReleaseStatus.Active && item.portals && item.portals.web_portal && item.portals.web_portal[0]"
+          <div class="ports" >
+		  <span class="ports-text"> {{ appService.getPorts(item) | lowercase }}</span>
+          </div>
+		  <div *ngFor="let portal of item.portals | keyvalue">
+          <button mat-button (click)="portalLink(item, portal.key )" *ngIf="item.status === ChartReleaseStatus.Active"
               ix-auto ix-auto-type="button" ix-auto-identifier="{{ item.name}}-portal">
-            {{ 'Portal' | translate }}
+            {{ portal.key | translate }}
```

Comment:
`| translate` is unnecessary, as translation system won't be able to pick up dynamic text coming from api.

Reply:
There is a `snakeToPascal` function in `EntityUtils` that you can use to have something `Web Portal` instead of `Web_portal` for now.

Reply:
I wasn't sure if it would translate or not, will change :)

That snakeToPascal function is cool though, thanks for the tip :)

Reply:
Worked like a charm, though snakeToPascal just removes _, I added a function snakeToHuman, that replaces _ with space. As I think that might be usefull in more places :)